materials/section-0/README.md

```diff
@@ -274,7 +237,134 @@ A `let` can easily be used in conjunction with `defn`
 (fn-with-locals 5) ;; => 1 2 7
 ```
 
-And that's it! We covered the bare basics of the language! Hopefully you understand how Clojure works and how to write basic expressions in Clojure. Hopefully you also have a grasp on the different *types* in Clojure, and how the language is structured.
+As mentioned above, these are the most common *special forms*. There are of course a lot more. There is a link to all special forms in the reference section of this page. Up next is examples.
+
+## Examples
+
+Next we will be looking at very common functions, and we'll also be breaking them down to understand thoroughly how they work.
+
+### Plus
+
+```clojure
+(+ 1 2) ;; => 3
+(+ 1) ;; => 1
+(+) ;; => 0
+(+ 2 2 2 2 2) ;; => 10
+```
+
+The plus `+` function takes 0 to infinite amount of numbers as arguments and sums them.
+
+All arithmetic functions (- * / +) work the same way. They all take between 0 or 1 to infinite, and performs their respective arithmetic operation on the given arguments.
```

Comment:
How about negative numbers? Surely they take between negative infinity to positive infinity? Or am I reading this incorrectly?

Also "infinite" should be changed to "infinity". Infinity is the number (if you call it that), to call something infinite is to say it is endless (sort of).

Yeah, I don't understand English either

Reply:
oh I totally misread that. You might want to change it to "They all take between 0 or 1 to infinite **arguments**" just to be super explicit.

Reply:
Good point! Will fix!